shell/src/main/java/alluxio/master/journal/tool/AbstractJournalDumper.java

```diff
@@ -105,9 +108,23 @@ private void readCompoundCheckpoint(CheckpointInputStream checkpoint, Path path)
   }
 
   private void readRocksCheckpoint(CheckpointInputStream checkpoint, Path path) throws IOException {
-    TarballCheckpointFormat.TarballCheckpointReader reader =
-        new TarballCheckpointFormat.TarballCheckpointReader(checkpoint);
-    reader.unpackToDirectory(path);
+    // Create RocksInodeStore over checkpoint stream for extracting the inodes.
+    try (PrintStream out =
+        new PrintStream(new BufferedOutputStream(new FileOutputStream(path.toFile())))) {
+      // Create an empty dir for storing the db.
+      Path dbPath = Paths.get(path.toFile().getPath() + "db");
```

Comment:
Is a "db" a sub directory of the path? Is there always a path separator before "db"?

Also, "db" should probably be more descriptive, like "rocksdb-checkpoint" or something.

Reply:
`path` will be used to create a file. `dbPath` will create a dir along with `path` in order to temporarily store the db content.